js/components/Todo/TodoList.js

```diff
@@ -0,0 +1,149 @@
+                                            <option value="NONE" ${todo.priority === priority.NONE ? 'selected' : ''}>순위</option>
+                                            <option value="FIRST" ${todo.priority === priority.FIRST ? 'selected' : ''}>1순위</option>
+                                            <option value="SECOND" ${todo.priority === priority.SECOND ? 'selected' : ''}>2순위</option>
+                                        </select>
+                                    </div>
+                                    ${todo.contents}
+                                </label>
+                                <button class="destroy" data-member-id="${memberId}" data-todo-id="${todo._id}"></button>
+                            </div>
+                            <input class="edit" value="할일을 입력해주세요."  data-member-id="${memberId}" data-todo-id="${todo._id}"/>
+                        </li>`
+            }).join('');
+        }
+        return '할일을 입력해주세요.';
+    }
+
+    constructor(element) {
+        super({
+            store,
+            element
+        });
+
+    }
+
+    render() {
+
+        this.element.querySelectorAll('.todoapp-container').forEach((node) => {
+            const nodeId = node.dataset.memberId;
+            const memberIdx = store.state.selectedTeam.members.findIndex((item) => nodeId === item._id);
+            const selectedMember = store.state.selectedTeam.members[memberIdx];
+            const $ul = node.querySelector('.todo-list');
+            const template = this.todoListTemplate(selectedMember.filterType, nodeId, selectedMember);
+            $ul.innerHTML = template;
+
+        })
+
+
+        this.element.querySelectorAll('.chip.select').forEach(async (node) => {
+            node.addEventListener(eventType.CHANGE, async ({key, target}) => {
+
+                const memberId = node.dataset.memberId;
+                const todoId = node.dataset.todoId;
+
+                const priorityName = target.value;
+                const response = await putTeamToMemberToTodoItemToPriority(store.state.selectedTeam._id, memberId, todoId, priorityName);
+                const changedTodoItemPriority = {
+                    memberId,
+                    todoList: response
+                }
+                store.dispatch('putMemberTodoListPriority', changedTodoItemPriority);
+
+            })
+        })
+        this.element.querySelectorAll('.toggle').forEach(async (node) => {
+            node.addEventListener(eventType.CHANGE, async ({key, target}) => {
+
+                const memberId = node.dataset.memberId;
+                const todoId = node.dataset.todoId;
+
+                const response = await putTeamToMemberToTodoItemToToggle(store.state.selectedTeam._id, memberId, todoId);
+                const changedTodoItemToggle = {
+                    memberId,
+                    todoList: response
+                }
```

Comment:
```suggestion
                const todoList = await putTeamToMemberToTodoItemToToggle(store.state.selectedTeam._id, memberId, todoId);
                const changedTodoItemToggle = {
                    memberId,
                    todoList,
                }
```
이렇게 쓰면 더 간편할 수 있습니다!